src/injected/components/details-dialog.tsx

```diff
@@ -70,6 +71,7 @@ export class DetailsDialog extends React.Component<DetailsDialogProps, DetailsDi
     public isBackButtonDisabled: () => boolean;
     public isNextButtonDisabled: () => boolean;
     public isInspectButtonDisabled: () => boolean;
+    public showInspectButtonMessage: () => boolean;
```

Comment:
I don't see any usage of `showInspectButtonMessage` in this class. Is this for the only purpose to call it from a test?
I can see we have this pattern on this class already but it would be nice if we can get off it.
It looks like it make sense to move the buttons out of this component, make a new small component with them. It also looks like this issue is a great opportunity to do that refactor.
Thoughts?

Reply:
I agree with what you are saying regarding the refactor; but I would rather get this issue done with  minimal changes first and then take refactoring the `details-dialog` as a later PR.
For `showInspectButtonMessage` this is used to check if initially we want to show the `inspect html error` or not in function `renderInspectMessage`

Reply:
So are you fine putting more tech debt on an already dirty component? I think I'll do the refactor first and only then do the fix. That should be our normal mode of operation.

As per `showInspectButtonMessage`... it's not being used on 'renderInspectMessage'... what we're using there on this PR is:
```typescript
if (this.props.dialogHandler.shouldShowInspectButtonMessage(this)) {
```
basically, using the `dialogHandler` directly, and thus, more clean up to do in the future.

Reply:
I never said that I am fine with adding more tech debt. I will take care of it but I was willing to delay it after we get this in. On `showInspectButtonMessage` i followed whatever pattern we were already using in this class.

Reply:
`showInspectButtonMessage` is not being called, only assign!

Reply:
will make the changes.

Reply:
Removed the `showInspectButtonMessage`.